tools/evmc/main.cpp

```diff
@@ -48,7 +48,7 @@ int main(int argc, const char** argv)
     std::string vm_config;
     std::string code_arg;
     int64_t gas = 1000000;
-    auto rev = EVMC_ISTANBUL;
+    auto rev = EVMC_MAX_REVISION;
```

Comment:
Here we rather want to go with stable Berlin for now. If we want to minimize number of changes here, we may want to introduce `EVMC_LATEST_STABLE_REVISON` (something we considered before). Comments @axic ?

Reply:
Took a bit of digging, but this was the issue: #175. And this was discussion sparking it: https://github.com/ethereum/evmc/pull/174#discussion_r245318043

I think what we discussed there still stands.